tests/test_utils.py

```diff
@@ -303,3 +306,437 @@ def check_multistream_classic(fnames):
                     actual, expected, decimal=4,
                     err_msg='Variable=%s, freq=%s, how=%s: '
                             'failed comparison' % (key, freq, how))
+
+def tsplit(string, delimiters):
+    '''Behaves like str.split but supports multiple delimiters.
+
+    '''
+
+    delimiters = tuple(delimiters)
+    stack = [string,]
+
+    for delimiter in delimiters:
+        for i, substring in enumerate(stack):
+            substack = substring.split(delimiter)
+            stack.pop(i)
+            for j, _substring in enumerate(substack):
+                stack.insert(i+j, _substring)
+
+    return stack
```

Comment:
can you give me an example usage of this function?  I'm thinking it is more complicated than it needs to be.